Projects/App/Sources/UI/Place/AddPlace/View/Views/PlaceCard.swift

```diff
@@ -0,0 +1,209 @@
+//
+//  PlaceCard.swift
+//  App
+//
+//  Created by Chanhee Jeong on 2022/10/28.
+//  Copyright © 2022 com.zesty. All rights reserved.
+//
+
+import UIKit
+import DesignSystem
+import SnapKit
+
+final class PlaceCardView: UIView {
+    
+    private let isSE = UIScreen.main.isLessThan376pt && !UIDevice.current.hasNotch
```

Comment:
p4;

이거 제가 프로필화면 코드리뷰하면서 써봤는데 hasNotch 안되는거 같더라고요 mini에서도 isSE=true 되는듯
그래서 UIScreen에 isHeightLessThan670pt 만들었습니다.
높이로 하면 한 번에 아이폰 SE인지 아닌지 확인 가능! 이거 쓰시죠

```suggestion
    private let isSE = UIScreen.main.isHeightLessThan670pt
```

Reply:
수정완료!

Reply:
5732fa2